presets/no-esm.json

```
@@ -42,14 +54,24 @@
       "matchPackageNames": ["@octokit/plugin-throttling", "@octokit/request"]
     },
     {
-      "allowedVersions": "<15.0.0",
+      "allowedVersions": "<13.0.0",
       "matchDatasources": ["npm"],
-      "matchPackageNames": ["@octokit/graphql-schema"]
+      "matchPackageNames": ["@octokit/webhooks"]
     },
     {
-      "allowedVersions": "<13.0.0",
+      "allowedVersions": "<13.2.3",
```

```suggestion
      "allowedVersions": "/(13\.2\.3/|13\.3\.2)",
```
There seems to be a newer CJS release already
[13.3.2](https://github.com/octokit/plugin-rest-endpoint-methods.js/compare/v13.3.1...v13.3.2-cjs.1)

---

the version `13.3.2-cjs.1` raises issues when running a `tsc:full` (tsc --skipLibCheck false --incremental false) that's why I didn't added it but I might be too strict

---

I have tried this with my [Backstage plugins repo](https://github.com/secustor/backstage-plugins/) and it throws errors regarding missing endpoints in both cases. 

AFAIK this is related to `@octokit/types` and probably this release https://github.com/octokit/types.ts/releases/tag/v13.5.0

If you find the exact release which should work for all or most users we can pin it, but I have to failed to find it in the past. https://github.com/backstage/backstage/pull/29479

The below pinning has not worked for me as well.

---

I have tried to on your Backstage plugins repo and it require to enforce resolution to theses versions to resolve issues from tsc

```json
  "resolutions": {
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "@octokit/plugin-rest-endpoint-methods": "13.2.2",
    "@octokit/plugin-paginate-rest": "11.3.1",
    "@octokit/types": "13.6.3"
  }
```

I'll remove them from this pr as this require extra configuration to make it work